extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -0,0 +1,269 @@
+  color: black !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device svg {
+  fill: var(--i-amphtml-story-dev-tools-accent) !important;
+  background: black !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device:hover, .i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device:focus {
+  background: var(--i-amphtml-story-dev-tools-accent-light) !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-bg {
+  position: fixed !important;
+  left: 0 !important;
+  top: 0 !important;
+  height: 100% !important;
+  width: 100% !important;
+  background: transparent !important;
+  display: flex !important;
+  justify-content: center !important;
+  align-items: center !important;
+  z-index: 1 !important;
+  transition: background 0.2s var(--i-amphtml-story-dev-tools-material-easing) !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-container {
+  background: black !important;
+  width: min(650px, 90%) !important;
+  border-radius: 10px !important;
+  padding: 40px !important;
+  position: relative !important;
+  transition: transform 0.2s var(--i-amphtml-story-dev-tools-material-easing), opacity 0.2s var(--i-amphtml-story-dev-tools-material-easing) !important;
+  transform: translateY(5vh) !important;
+  opacity: 0 !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-bg[active] {
+  background: var(--i-amphtml-story-dev-tools-device-gray-transparent-1) !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-bg[active] .i-amphtml-story-dev-tools-device-dialog-container  {
+  transform: translateY(0) !important;
+  opacity: 1 !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-container h1 {
+  font-size: 12px !important;
+  text-transform: uppercase !important;
+  opacity: .4 !important;
+  letter-spacing: 1px !important;
+  font-weight: normal !important;
+  margin: 0px !important;
+  margin-bottom: 5px !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-container h1:not(:first-child) {
+  margin-top: 25px !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-container.i-amphtml-story-dev-tools-device-dialog-add-devices h1 {
+  margin-inline-start: 15px !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-link {
+  color: var(--i-amphtml-story-dev-tools-device-white-transparent-3) !important;
+  text-decoration: none !important;
+  display: block !important;
+  padding: 10px 0 !important;
+  transition: color 0.2s var(--i-amphtml-story-dev-tools-material-easing) !important;
+  display: flex !important;
+  align-items: center !important;
+  justify-content: space-between !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-link svg {
+  vertical-align: middle !important;
+  opacity: 0.6 !important;
+  transition: opacity 0.2s var(--i-amphtml-story-dev-tools-material-easing), transform 0.2s var(--i-amphtml-story-dev-tools-material-easing) !important;
```

Comment:
> Elements that begin and end at rest use standard easing.

Reply:
This is standard, will update the variable name